samples/grpc/shopping-analytics-service-java/src/main/resources/persistence.conf

```diff
@@ -7,19 +8,13 @@ akka {
       plugin = "akka.persistence.r2dbc.snapshot"
     }
     r2dbc {
-      # yugabyte or postgres
-      dialect = "postgres"
       connection-factory {
-        driver = "postgres"
-
         host = "localhost"
         host = ${?DB_HOST}
-
         port = 5432
         database = "postgres"
         user = "postgres"
         user = ${?DB_USER}
-
         password = "postgres"
         password = ${?DB_PASSWORD}
```

Comment:
Just a note: All these are defaults so it would also be possible to just add the maybe-env-entries and use the others from default config.

Reply:
um maybe, I think it's fine to repeat them for clarity here
e.g. I see what I can use with my psql without having to look at reference.conf